prerendering.bs

```diff
@@ -658,6 +673,41 @@ The <dfn attribute for="PerformanceNavigationTiming">activationStart</dfn> gette
 
 1. Return a {{DOMHighResTimeStamp}} with a time value equal to the [=current document=]'s [=Document/activation start time=].
 
+<h3 id="client hint cache">Interaction with Client Hint Cache</h3>
+
+We need to ensure that the [=Accept-CH cache=], which is owned by user agent as a global storage, is not modified while prerendering, but is properly updated after activation. The following modifications ensure this:
+
+Each [=prerendering browsing context=] maintains its <dfn for="prerendering browsing context">prerender-scoped Accept-CH cache</dfn>, to persist the origin's opt-in to receive the indicated header fields.
+
+The [=prerendering browsing context/prerender-scoped Accept-CH cache=] is an [=ordered map=] of [=origins=] to [=client hints sets=].
+
+<div algorithm="add a new prerender Accept-CH cache entry">
+
+Similar to [=add a new Accept-CH cache entry=], to <dfn>add a new prerender Accept-CH cache entry</dfn> to the [=prerendering browsing context/prerender-scoped Accept-CH cache=],
+given a [=prerendering browsing context=] |bc|, an |origin| and a [=client hints set=] |hintSet|,
+[=map/set=] |bc|'s [=prerendering browsing context/prerender-scoped Accept-CH cache=][|origin|] to |hintSet|.
+
+</div>
+
+<div algorithm="append client hints to request patch">
+  Modify the <a spec=CLIENT-HINTS-INFRASTRUCTURE>append client hints to request</a> algorithm, by prepending the following steps before iterating over the |hintSet|:
+
+  1. Let |browsingContext| be |settingsObject|’s [=environment settings object/global object=]'s [=Window/browsing context=].
+    If |browsingContext| is a [=prerendering browsing context=], then:
+      1. Let |orign| be |request|'s [=request/url=]'s [=url/origin=].
+      1. Set |hintSet| to the [=set/union=] of |hintSet| and |browsingContext|'s [=prerendering browsing context/prerender-scoped Accept-CH cache=][|origin|].
+  
+</div>
+
+<div algorithm="update the Client Hints set patch">
+  Modify the <a spec=CLIENT-HINTS-INFRASTRUCTURE>update the Client Hints set</a> algorithm, by updating the last step.
+
+  1. If |browsingContext| is a [=prerendering browsing context=],
+    [=add a new prerender Accept-CH cache entry=] with |response|'s [=response/url=]'s [=url/origin=] and |settingsObject|'s [=environment settings object/client hints set=].
```

Comment:
This is missing passing |browsingContext|, now that "add a new prerender Accept-CH cache entry" takes three arguments

Reply:
ooops.. 💯 right, I fixed it. 
Thanks!!